src/libslic3r/PerimeterGenerator.cpp

```diff
@@ -283,6 +332,22 @@ static ExtrusionEntityCollection traverse_loops(const PerimeterGenerator &perime
             BoundingBox bbox(polygon.points);
             bbox.offset(SCALED_EPSILON);
 
+            // Always reverse extrusion if use fuzzy skin: https://github.com/SoftFever/OrcaSlicer/pull/2413#issuecomment-1769735357
```

Comment:
Question: why are we differenciate contour and hole for reversing?

Reply:
To improve surface quality when only contour/hole surface has steep overhang.

Reply:
make sense 👍